src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2874,6 +2876,8 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/StorageResourceModel'
+        securityClassification:
+          $ref: '#/components/schemas/DatasetSecurityClassification'
```

Comment:
Why do we need this in both the DatasetModel and the DatasetSummaryModel (same question for the equivalent snapshot models). Would the dataset model not be enough?